Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/OwnershipSecurity.ts

```diff
@@ -44,6 +44,10 @@ export async function rejectByOwnershipSecurity(
   try {
     const result: QueryResult = await client.query(findOwnershipForDocumentSql(id));
```

Comment:
I was able to see the same error as shown in screenshot above when I put a breakpoint here.